src/v3/src/components/InfoBox/InfoBox.tsx

```diff
@@ -51,7 +63,9 @@ const InfoBox: UISchemaElementComponent<{
         {...({ 'data-se': dataSe })}
         className={`infobox-${messageClass.toLowerCase()}`}
       >
-        {message}
+        { title && <AlertTitle>{title}</AlertTitle> }
```

Comment:
Since this is added, I think you can remove the `title` property from line 61 right?

Reply:
Yeah I think this can be removed.  I just wasnt sure if it was still needed for accessibility reasons.